### ISSUE TITLE
Using button colors from editor adds unwanted padding and broken css property to the class in universal.css

### ISSUE THREAD

**Issue by lucianahanan** [soflyy/oxygen-bugs-and-features#3256]
1. Add a button
2. Add a class named button-red
3. Put colors for the normal and hover state ( in a way so the off state has solid color/white letter, and the hover state has it inverted
( by the way I wanted to have solid on off, and outline on hover, but it seems to not accept that, it changes for both )
4. Oxygen adds a padding 10px 16px  to the class, even if I put nothing in the class paddings or in custom css. This overrides any custom padding in the class too, in a way I can't just have paddings.
5. It also adds a broken property named buton-hover_color to the class.

![image](https://user-images.githubusercontent.com/1200923/222903831-5515b316-d418-4b19-a55d-b33f874f4e97.png)
![image](https://user-images.githubusercontent.com/1200923/222903857-b1c2aed2-76b3-4ccc-b976-7b32294c9625.png)
![image](https://user-images.githubusercontent.com/1200923/222903866-33ddce1d-bf75-43ee-8c85-d13ad2973690.png)

A friend added the button with a class the same way I did, and the same happened to the end of the oxygen's css:

https://oxygen-qfdxm3hc4xfrl.oxygen-demo.qsandbox.org/wp-content/uploads/oxygen/css/universal.css?cache=1677909990&ver=6.1.1&fbclid=IwAR0L9aNcCxsE6bD4EuRsK1q9b0db5nYr6ZxBMauj6DirTHF1YW7Ugm2fStg

**Comment by briandash2** [comment]
I don't think this is a bug. Clicking the "Solid" or "Outline" buttons in the Primary panel IS adding style presets to the button. If you don't click the "Solid" or "Outline" you can easily add a background color + border of the same color to the button. Set the padding to whatever you want. Then change the state to hover. Change the bg color to transparent. Then change the font color. Works perfectly every time.
